src/util/snipe.py

```diff
@@ -0,0 +1,29 @@
+import discord
+
+from src import config as conf
+
+def timestamp(msg: discord.Message):
+    return msg.edited_at if msg.edited_at is not None else msg.created_at
+
+def into_embeds_chunks(history: list[discord.Message]):
+    author = history[0].author.display_name
+    avatar = history[0].author.display_avatar
+
+    embeds: list[tuple[int, discord.Embed]] = []
+    for state in reversed(history):
+        embed = discord.Embed(description=state.content, timestamp=timestamp(state)).set_author(name=author, icon_url=avatar)
```

Comment:
.display_avatar is optional, hence avatar might be `None`. what happens if the author has no avatar?